styles/learnxinyminutes/catppuccin.user.css

```diff
@@ -0,0 +1,177 @@
+    @sapphire: @catppuccin[ @@lookup][ @sapphire];
+    @blue: @catppuccin[ @@lookup][ @blue];
+    @lavender: @catppuccin[ @@lookup][ @lavender];
+    @text: @catppuccin[ @@lookup][ @text];
+    @subtext1: @catppuccin[ @@lookup][ @subtext1];
+    @subtext0: @catppuccin[ @@lookup][ @subtext0];
+    @overlay2: @catppuccin[ @@lookup][ @overlay2];
+    @overlay1: @catppuccin[ @@lookup][ @overlay1];
+    @overlay0: @catppuccin[ @@lookup][ @overlay0];
+    @surface2: @catppuccin[ @@lookup][ @surface2];
+    @surface1: @catppuccin[ @@lookup][ @surface1];
+    @surface0: @catppuccin[ @@lookup][ @surface0];
+    @base: @catppuccin[ @@lookup][ @base];
+    @mantle: @catppuccin[ @@lookup][ @mantle];
+    @crust: @catppuccin[ @@lookup][ @crust];
+    @accent-color: @catppuccin[ @@lookup][ @@accent];
+
+    --ctp-rosewater: @rosewater;
+    --ctp-flamingo: @flamingo;
+    --ctp-pink: @pink;
+    --ctp-mauve: @mauve;
+    --ctp-red: @red;
+    --ctp-maroon: @maroon;
+    --ctp-peach: @peach;
+    --ctp-yellow: @yellow;
+    --ctp-green: @green;
+    --ctp-teal: @teal;
+    --ctp-sky: @sky;
+    --ctp-sapphire: @sapphire;
+    --ctp-blue: @blue;
+    --ctp-lavender: @lavender;
+    --ctp-text: @text;
+    --ctp-subtext1: @subtext1;
+    --ctp-subtext0: @subtext0;
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@lookup =latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    .share {
+      background-color: @mantle;
+      border-color: @crust;
+    }
+
+    .share:hover {
+      background-color: lighten(@mantle, 10%);
+    }
+
+    .theme-choice button {
+      color: @text;
+      background-color: @mantle;
+      border-color: @crust;
+    }
+
+    .theme-choice button:hover {
+      background-color: lighten(@mantle, 10%);
+    }
+
+    #lang-select {
+      color: @text;
+      background-color: @mantle;
+      border-color: @crust;
+    }
+
+    td.lang span a {
+      color: @text;
+      background-color: @mantle;
+      border-color: @crust;
+    }
+
+    td.lang span a:hover {
+      background-color: lighten(@mantle, 10%);
+    }
+
+    body {
+      background-color: @base;
+      color: @text;
+    }
+
+    td,
+    th {
+      color: @text;
+    }
+
+    footer {
+      p {
+        color: @subtext1;
+      }
+    }
+
+    a {
+      color: @accent-color !important;
+    }
+
+    hr {
+      border-color: @surface1;
+    }
+
+    code {
+      background-color: @mantle;
+      color: @text;
+      padding: 0.1em;
+      border-radius: 0.3em;
+      letter-spacing: 0.1em;
+      margin-left: 0.05em;
+      margin-right: 0.05em;
```

Comment:
Avoid changing non-color properties like this unless strictly required.
```suggestion
```

Reply:
I thought this looks better but I can see why it'd be best to have all userstyles *only* affect colors, otherwise we could argue for making these "tweaks" everywhere